configs/s2ef/200k/forcenet/fn_forceonly.yml

```diff
@@ -1,7 +1,7 @@
 trainer: forces
 
 dataset:
-  - src: data/s2ef/all/train/
+  - src: data/s2ef/200k/train/
   - src: data/s2ef/all/val_id/
```

Comment:
could we change the val one to 200k as well?

Reply:
There is no `200k` val - it's always in `all`

Reply:
Oh yeah, makes sense.